src/utils/storyblok.ts

```diff
@@ -1,9 +1,16 @@
 import { LinkComponent } from '../storyblok/StoryContainer'
 
-export const getStoryblokLinkUrl = (link: LinkComponent) =>
-  link.linktype !== 'story' || /^\//.test(link.cached_url)
-    ? link.cached_url
-    : `/${link.cached_url}`
+export const getStoryblokLinkUrl = (link: LinkComponent) => {
+  const cachedLink =
+    link.linktype !== 'story' || /^\//.test(link.cached_url)
+      ? link.cached_url
+      : `/${link.cached_url}`
+  const publicHost = getPublicHost()
+  return publicHost
+    ? cachedLink.replace('https://www.hedvig.com', publicHost)
```

Comment:
Should we skip the replace if it's in prod?

Reply:
I was thinking about it but we'll substitute `https://www.hedvig.com` with `https://www.hedvig.com` and i think it dosen't really matter right? Maybe it'd be sliiiiightly more efficient from a performance perspecitve but i don't think it'd be in any way noticable